public/data/projects.json

```diff
@@ -5,7 +5,7 @@
       "name": "CO2-Labels for University Canteen",
       "image": "/images/canteen.jpg",
       "organisation_name": "sneep Erlangen e.V.",
-      "organisation_image": "/images/sneep.png",
+      "organisation_image": "images/sneep.png",
       "location": "Erlangen, Germany",
```

Comment:
Should we keep `/` here? just like `image` key https://github.com/climateconnect/climateconnect/pull/81/files#diff-f2b2feedd7adbacbd6d0b89217a52686R6 here?

Reply:
This must have been a mistake when merging. Fixed it.